gears_cli/__main__.py

```diff
@@ -73,10 +93,20 @@ def print_res(res, res_id):
 @click.option('--port', default=6379, type=int, help='Redis port to connect to')
 @click.option('--user', default=None, help='Redis acl user')
 @click.option('--password', default=None, help='Redis password')
+@click.option('--ssl', default=False, type=bool, help='Use ssl')
```

Comment:
Why not just pull out the --ssl? I know that redis has this argument, but the second you specify any of the sub options (ssl-password for example) then clearly we're using SSL. Better user experience IMHO.